packages/strapi-admin/admin/src/translations/sv.json

```diff
@@ -0,0 +1,157 @@
+{
+  "Analytics": "Analytics",
+  "Content Manager": "Innehållshantering",
+  "Content Type Builder": "Innehållstyp Byggare",
+  "Email": "E-post",
+  "Files Upload": "Uppladdning av filer",
+  "HomePage.notification.newsLetter.success": "Prenumererade på nyhetsbrevet",
+  "New entry": "Nytt inlägg",
+  "Password": "Lösenord",
+  "Provider": "Leverantör",
+  "ResetPasswordToken": "Återställ lösenordstoken",
+  "Role": "Roll",
+  "Roles & Permissions": "Roller och behörigheter",
+  "Settings Manager": "Inställningshanterare",
+  "Username": "Användarnamn",
+  "Users": "Användare",
+  "Users & Permissions": "Användare och behörigheter,
+  "app.components.BlockLink.code": "Kodsexempel",
+  "app.components.BlockLink.code.content": "Lär dig genom att testa verkliga projekt utvecklade samhället.",
+  "app.components.BlockLink.documentation": "Läs dokumentationen",
+  "app.components.BlockLink.documentation.content": "Upptäck koncept, referensguider och handledning.",
+  "app.components.Button.cancel": "Avbryt",
+  "app.components.Button.save": "Spara",
+  "app.components.ComingSoonPage.comingSoon": "Kommer snart",
+  "app.components.ComingSoonPage.featuresNotAvailable": "Denna funktion är fortfarande under aktiv utveckling.",
+  "app.components.DownloadInfo.download": "Nedladdning pågår ...",
+  "app.components.DownloadInfo.text": "Det kan ta en minut. Tack för ditt tålamod.",
```

Comment:
You normally say "Tack för att du väntar" rather than "tack för ditt tålamod".

Reply:
> You normally say "Tack för att du väntar" rather than "tack för ditt tålamod".

Well both work, I personally hear "tack för ditt tålamod" more than "Tack för att du väntar".